cmd/podman/containers/run.go

```diff
@@ -148,6 +148,21 @@ func run(cmd *cobra.Command, args []string) error {
 		imageName = name
 	}
 
+	// If this is a checkpoint image, invoke container restore.
+	checkpointImage, err := utils.IsCheckpointImage(registry.GetContext(), []string{imageName})
+	if checkpointImage {
+		if err != nil {
```

Comment:
This seems like it should be outside of this block.  Err could happen if checkpointImage == false.

Reply:
When checkpointImage = false, `err` could be `"image is not a checkpoint"`.